docs/API.md

```diff
@@ -18,12 +18,16 @@ There are a number of methods that a DB storage backend should implement:
     * .createSessionToken(tokenId, sessionToken)
     * .updateSessionToken(tokenId, sessionToken)
     * .sessionToken(id)
+    * .sessionTokenVerified(tokenId)
```

Comment:
I agree with the naming concerns here.  Two pieces of prior art that feed into it:

* There's an existing `forgotPasswordVerified` method that is actually a state-changing method, to be called to indicate "this token was just verified".  The asymmetry in behaviour between similarly-named `forgotPasswordVerified` and `sessionTokenVerified` can only lead to trouble...
* There's an existing `sessionTokenWithDevice` method that verbosely but accurately says what it will do - it will fetch the session token and its additional device information

When https://github.com/mozilla/fxa-auth-server/pull/1232 is stable, we should take a step back and see how often we'll be loading these tokens with and without their verification status.  It may be that the verification-status-less version is almost never called in practice so there's no need to optimize for it.